store/entity-store.html

```diff
@@ -160,9 +160,11 @@
 							}
 							return Promise.reject(response.status);
 						}.bind(this))
-						.then(function(body) {
-							const entity = window.D2L.Hypermedia.Siren.Parse(body);
-							this.update(entityId, resolved, entity);
+						.then(window.D2L.Hypermedia.Siren.Parse)
```

Comment:
this wasn't a promise before, doesn't this need to be wrapped in a promise to become `then`-able?

Reply:
Nope, this would just pass the parsed entity to the next function, which you then return a promise and then the `then`ing proceeds. Makes sense